script/brave_license_helper.py

```diff
@@ -1,7 +1,7 @@
-# Copyright (c) 2019 The Brave Authors. All rights reserved.
+# Copyright (c) 2016 The Brave Authors. All rights reserved.
```

Comment:
@fmarier can you take a look at the changes happening here?

Reply:
Hm, that's a bit weird. 2019 is the correct date.
```suggestion
# Copyright (c) 2019 The Brave Authors. All rights reserved.
```